react/src/pages/ChecksPage.js

```diff
@@ -21,116 +21,43 @@ import Yaml from '../utils/Yaml';
 import LiveTime from '../LiveTime';
 import Styles from '../Styles';
 
-const ChecksPage = (props) => {
```

Comment:
I'm afraid a ton has changed here mostly because I had to lift a bunch of functions/component, which we nested, out to the top-level, as nested components I've learned can be problematic re-rendering-wise, and also as a step toward factoring out eventually into (reusable) separate modules. The main functional thing added here is related to actions support.